lib/diplomat/autopilot.rb

```diff
@@ -9,6 +9,10 @@ class Autopilot < Diplomat::RestClient
     # @param options [Hash] options parameter hash
     # @return [OpenStruct] all data associated with the autopilot configuration
     def get_configuration(options = {})
+      custom_params = []
+      custom_params << use_named_parameter('dc', options[:dc]) if options[:dc]
+      custom_params << use_named_parameter('stale', options[:dc]) if options[:stale]
```

Comment:
This is not `options[:dc]` and should be stale furthermore : stale is already parsed in `parse_options` which is called by `send_get_request`.
So, you just don't this line